### PR TITLE
Translated email templates to Swedish

### LINKED ISSUE #26: Write swedish email templates
The current email templates are in english.

### PR DESCRIPTION
**Pull request by Greenheart**
closes #26 

The build is failing because of a **lack of environment variables** in the CI-tests. How should that be fixed?

### CODE REVIEW

**Comment by hugolundin** [comment]
"svar ej blir lästa" is wrong right? 😅

**Comment by Greenheart** [comment]
Yes there might be some mistakes so please comment on the specific line in
the diff where you find them! :)

torsdag 21 april 2016 skrev Hugo Lundin notifications@github.com:

**Comment by hugolundin** [comment]
Did not know about this function. Awesome, will do!

File: web/app/templates/auth/email/change_email.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+<p>Kära {{ user.username }},</p>
+<p>Vänligen <a href="{{ url_for('auth.change_email', token=token, _external=True) }}">klicka här</a> för att bekräfta din nya e-postadress.</p>
+<p>Alternativt så kan du klistra in den följande länken i webbläsarens adressfält:</p>
 <p>{{ url_for('auth.change_email', token=token, _external=True) }}</p>
-<p>Sincerely,</p>
-<p>The Flasky Team</p>
-<p><small>Note: replies to this email address are not monitored.</small></p>
+<p>Hälsningar,</p>
+<p>Chronos Development Team</p>
+<p><small>Observera: svar till denna e-postadress ej blir lästa.</small></p>
```
Comment (hugolundin) [inline web/app/templates/auth/email/change_email.html]: This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".

File: web/app/templates/auth/email/change_email.txt [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 
-Sincerely,
+Hälsningar,
+Chronos Development Team
 
-The Flasky Team
-
-Note: replies to this email address are not monitored.
\ No newline at end of file
+Observera: svar till denna e-postadress ej blir lästa.
```
Comment (hugolundin) [inline web/app/templates/auth/email/change_email.txt]: This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".

File: web/app/templates/auth/email/login_email.html [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
+<h1>Kära, {{ user.first_name }} {{ user.last_name }}!</h1>
+<p>Detta mail innehåller en länk som låter dig logga in i Chronos.</p>
+<p><a href="{{ url_for('auth.login_email', token=token, _external=True) }}">Klicka här för att logga in</a></p>
+<p>Om länken inte fungerar, vänligen klistra in följande länk i webbläsarens adressfält:</p>
 <p>{{ url_for('auth.login_email', token=token, _external=True) }}</p>
-<p>Sincerly, Chronos Development Team.</p>
\ No newline at end of file
+<p>Hälsningar,</p>
+<p>Chronos Development Team</p>
+<p><small>Observera: svar till denna e-postadress ej blir lästa.</small></p>
```
Comment (hugolundin) [inline web/app/templates/auth/email/login_email.html]: This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".

File: web/app/templates/auth/email/login_email.txt [hunk 4]
```diff
@@ -1,4 +1,11 @@
...
+
+Detta mail innehåller en länk som låter dig logga in i Chronos.
+
+Vänligen klistra in följande länk i webbläsarens adressfält:
+{{ url_for('auth.login_email', token=token, _external=True) }}
+
+Hälsningar,
+Chronos Development Team
+
+Observera: svar till denna e-postadress ej blir lästa.
```
Comment (hugolundin) [inline web/app/templates/auth/email/login_email.txt]: This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".

File: web/app/templates/auth/email/reset_password.html [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
 <p>{{ url_for('auth.password_reset', token=token, _external=True) }}</p>
-<p>If you have not requested a password reset simply ignore this message.</p>
-<p>Sincerely,</p>
-<p>The Flasky Team</p>
-<p><small>Note: replies to this email address are not monitored.</small></p>
\ No newline at end of file
+<p>Om du inte vill byta lösenord kan du ignorera detta meddelande.</p>
+<p>Hälsningar,</p>
+<p>Chronos Development Team</p>
+<p><small>Observera: svar till denna e-postadress ej blir lästa.</small></p>
```
Comment (hugolundin) [inline web/app/templates/auth/email/reset_password.html]: This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".

File: web/app/templates/auth/email/reset_password.txt [hunk 6]
```diff
@@ -1,13 +1,11 @@
...
-Sincerely,
+Om du inte vill byta lösenord kan du ignorera detta meddelande.
 
-The Flasky Team
+Hälsningar,
+Chronos Development Team
 
-Note: replies to this email address are not monitored.
\ No newline at end of file
+Observera: svar till denna e-postadress ej blir lästa.
```
Comment (hugolundin) [inline web/app/templates/auth/email/reset_password.txt]: This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".

File: web/app/templates/auth/email/login_email.html [hunk 7]
```diff
@@ -1,6 +1,8 @@
-<h1>Hello, {{ user.first_name }} {{ user.last_name }}!</h1>
-<p>This email contains a link for you to sign in to chronos.</p>
-<p><a href="{{ url_for('auth.login_email', token=token, _external=True) }}">Click here to sign in</a></p>
-<p>If the link does not work, please copy and paste this in your browser url bar:</p>
+<h1>Kära, {{ user.first_name }} {{ user.last_name }}!</h1>
+<p>Detta mail innehåller en länk som låter dig logga in i Chronos.</p>
+<p><a href="{{ url_for('auth.login_email', token=token, _external=True) }}">Klicka här för att logga in</a></p>
+<p>Om länken inte fungerar, vänligen klistra in följande länk i webbläsarens adressfält:</p>
```
Comment (hugolundin) [inline web/app/templates/auth/email/login_email.html]: Double usage of "länk". Should probably be "Om länken inte fungerar, vänligen klistra in följande i webbläsarens adressfält:"

**Comment by Greenheart** [comment]
PR updated.

Is there any other files that should be translated to close this issue?

**Pull request merged**